common/src/main/java/org/apache/celeborn/common/network/client/TransportClient.java

```diff
@@ -338,4 +352,26 @@ protected void handleFailure(String errorMsg, Throwable cause) {
       callback.onFailure(new IOException(errorMsg, cause));
     }
   }
+
+  private class RpcChannelListenerWithCompleteCallback extends RpcChannelListener {
+    final BooleanSupplier closeCallBack;
```

Comment:
No result is expected, the supplier can be replaced.

Reply:
yes, runnable is more  appropriate